python-survey-app/remove_empty_record.py

```diff
@@ -0,0 +1,9 @@
+def remove_empty_records(dataset):
+
+    results = []
+
+    for rec in dataset:
+        if all(rec):
+            results.append(rec)
+
+    return results
```

Comment:
There is no new line at the end of this def.

Should be there fore readability purposes.